pom.xml

```diff
@@ -140,12 +159,12 @@
     </profiles>
 
     <build>
+        <finalName>app</finalName>
```

Comment:
I got this issue: https://github.com/spring-projects/spring-boot/issues/25590

I removed from Spring Boot Plugin section...

> The name of the generated artifact should be configured in the <build> section of the POM, not in the Spring Boot plugin configuration. That is, it should be:

Reply:
that's ok/.